frontend/src/App.tsx

```diff
@@ -13,9 +15,12 @@ function App() {
         <Router>
             <div className="App">
                 <Switch>
-                    <Route path="/login" component={Login} />
-                    <Route path="/registration" component={Registration} />
-                    <Route path="/email-confirmation/:id" component={EmailConfirmation} />
+                    <Authentication>
+                        <Route path="/" component={Home} />
```

Comment:
specifying only `path="/"` will match every route.
For example `/login` will also be matched, as it starts with `/`.
You can see the effect here, see `Home` being displayed at the top.
![image](https://user-images.githubusercontent.com/5684735/115409328-b2e63e00-a202-11eb-828a-0e92349aade0.png)

For this not to happen you can add `exact` prop to this route definition:
https://reactrouter.com/web/api/NavLink/exact-bool